packages/flutter_tools/lib/src/commands/analyze_continuously.dart

```diff
@@ -91,6 +96,20 @@ class AnalyzeContinuously extends AnalyzeBase {
         }
       }
 
+      // Summarize dartdoc issues rather than displaying each individually
+      int membersMissingDocumentation = 0;
+      if (flutterRepo && !showDartDocIssuesIndividually) {
+        errors.removeWhere((AnalysisError error) {
```

Comment:
I modified the code to create a separate `detailedErrors` list as you suggested.